forecast/paper/response-to-reviewers.tex

```diff
@@ -0,0 +1,304 @@
+
+  We also have strong reasons for not using model-based approaches (one being the
+  nonstationarity the Reviewer mentions above, but also the belief that these
+  models are wrong). We felt that the related investigation in 2B was more
+  pertinent.
+
+  To respond more directly:
+
+  $\bullet$ We have added a section to the Supplement (and point
+  to it around 
+  line 85 in the main text), that performs a Sign Test for differences in
+  accuracy. P-values there tend to cluster near zero suggesting again that these
+  improvements are statistically significant.
+
+  $\bullet$ We also use a Diebold-Mariano test
+  for comparing forecast accuracy. The results for this test are more mixed,
+  likely due to nonstationarity the Reviewer mentions above (the indicators are
+  more helpful in some periods than others, see also the upswing/downswing
+  discussion). 
+\end{resp}
+
+3. Are the results in this paper practically significant? I see no discussion of
+this question, which seems unbelievable for me to be saying, given that the
+authors refuse to consider statistical significance. And yet here I am.
+
+\begin{resp}
+  See also Reviewer 1 point 3. And the general comment below (Point 8).
+  Practically, on average, if our goal is to predict 11 days ahead, when the AR
+  model is at its best, CHNG-CLI
+  doesn’t help (at least with these simple models). However, CHNG-COVID,
+  DV-CLI, and Google-AA ``buy'' 4 extra days for the 
+  same accuracy. CTIS-CLIIC ``buys'' about 5 extra days. What we mean here is
+  that if we have a certain tolerance for inaccuracy, adding these signals increases the
+  horizon at which we can predict within our tolerance. As for the
+  practical significance of 4-5 extra days, this would of course depend on the use case.  However, cases grow
+  exponentially, so making public policy changes 15 or 16 days ahead rather than
+  11 can mean real gains. There also seems to be a fairly constant mapping
+  between cases and future hospitalizations. Giving
+  hospitals a few extra days to accumulate supplies and find personnel is
+  meaningful. For predicting hotspots, we see similar results (gains of a few
+  days) though with different indicators. 
+\end{resp}
+
+4. For the cumsum plots I mentioned above, and other related techniques, we can
+plot all the model prediction error cumsum curves versus time and thereby
+compare different models for size, nonstationarity and significance of
+prediction improvement. The plots that have been presented in Figure 4 say do
+none of those tasks.
+
+\begin{resp}
+  TODO Any ideas here? It's not clear to me that this is different from above.
+\end{resp}
+
+5. The idea of using Asof data is of course important but I have known about it
+for twenty years and practiced it routinely; it has apparently been in routine
+use in econometrics for 40 years.
+
+\begin{resp}
+  We appreciate the reviewer's perspective on the ubiquity of Asof data in
```

Comment:
I prefer "the reviewer" to "the Reviewer", but in either case we should be consistent (elsewhere it was "the Reviewer")